packages/recomponents/src/components/r-pagination/r-pagination.md

```diff
@@ -1 +1,5 @@
-Pagination component can be used to split views or data collections with ability to navigate in this set
\ No newline at end of file
+Pagination component can be used to split views or data collections with ability to navigate in this set.
+
+Depending on the parent's component width and / or number of available provided pages, the pagination component will automatically scale if required, some pages at the begin / at the end of list will be hidden with `…` symbol.
+
+New version require `v-model` to manipulate current page.
```

Comment:
does this mean v-model is always required for usage?

Reply:
Yes, like in [Vuetify](https://v15.vuetifyjs.com/en/components/paginations).

I think it's possible to support both modes at the same time: v-model and regular prop to page initial page with $watch usage, but maybe this will be to overload.